charts/nexus-repository-manager/README.md

```diff
@@ -116,68 +116,71 @@ The command removes all the Kubernetes components associated with the chart and
+| `route.portName`                           | Target port name of service                                                                  | `docker`                                                                                                                                        |
+| `route.labels`                             | Labels to be added to route                                                                  | `{}`                                                                                                                                            |
+| `route.annotations`                        | Annotations to be added to route                                                             | `{}`                                                                                                                                            |
+| `route.path`                               | Host name of Route e.g jenkins.example.com                                                   | nil                                                                                                                                             |
+| `serviceAccount.create`                    | Set to true to create ServiceAccount                                                         | `true`                                                                                                                                          |
+| `serviceAccount.annotations`               | Set annotations for ServiceAccount                                                           | `{}`                                                                                                                                            |
+| `serviceAccount.name`                      | The name of the service account to use. Auto-generate if not set and create is true          | `{}`                                                                                                                                            |
+| `persistence.enabled`                      | Set false to eliminate persistent storage                                                    | `true`                                                                                                                                          |
+| `persistence.existingClaim`                | Specify the name of an existing persistent volume claim to use instead of creating a new one | nil                                                                                                                                             |
+| `persistence.storageSize`                  | Size of the storage the chart will request.                                                  | `8Gi`                                                                                                                                           |
```

Comment:
the last 3 persistence options are the only thing new to this table that was otherwise just reformatted.